src/frontends/onnx/frontend/src/op/com.microsoft/skip_simplified_layer_normalization.cpp

```diff
@@ -0,0 +1,136 @@
+﻿// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "core/operator_set.hpp"
+#include "exceptions.hpp"
+#include "openvino/frontend/exception.hpp"
+#include "openvino/op/add.hpp"
+#include "openvino/op/convert.hpp"
+#include "openvino/op/divide.hpp"
+#include "openvino/op/multiply.hpp"
+#include "openvino/op/range.hpp"
+#include "openvino/op/reduce_mean.hpp"
+#include "openvino/op/shape_of.hpp"
+#include "openvino/op/sqrt.hpp"
+#include "utils/common.hpp"
+
+using namespace ov::op;
+using ::ONNX_NAMESPACE::TensorProto_DataType;
+
+namespace ov {
+namespace frontend {
+namespace onnx {
+namespace com_microsoft {
+namespace opset_1 {\
+
+ov::OutputVector simplified_layer_normalization(const ov::frontend::onnx::Node& node) {
+    const auto inputs = node.get_ov_inputs();
+    const auto num_inputs = inputs.size();
+
+    FRONT_END_GENERAL_CHECK(num_inputs == 2,
```

Comment:
Hi, let's use common::default_op_checks(node, 2); which I had introduced some time ago, for example: https://github.com/openvinotoolkit/openvino/blob/e07546d6414768c7fea3769eef40bcb544e9352a/src/frontends/onnx/frontend/src/op/dequantize_linear.cpp#L205

Reply:
Done